met-web/src/utils/index.ts

```diff
@@ -81,3 +81,11 @@ export const isDarkColor = (color: string, sensitivity = 0.5) => {
     const luminance = (0.299 * r + 0.587 * g + 0.114 * b) / 255;
     return luminance < sensitivity;
 };
+
+export const generateUniqueKey = () => {
```

Comment:
As discussed a bit, let's review the use of this function as a "key" generator. There are other recommended methods for this